### PR TITLE
docs(guide & api): feedback/issue links for Dart should refer to Dart repos

### PR DESCRIPTION
**Pull request by chalin**
- Also added links to chapters mentioned in table.
- Other minor copyedits.

### CODE REVIEW

File: public/docs/ts/latest/guide/index.jade [hunk 1]
```diff
@@ -80,37 +81,28 @@ table(width="100%")
...
-
-  Use the [angular.io Github repo](https://github.com/angular/angular.io) for **documentation** issues and pull requests.
+  - The [Cheat Sheet](cheatsheet.html) lists Angular syntax for common scenarios.
+  - The [Glossary](glossary.html) defines terms that Angular developers should know.
+  - The [API Reference](../api/) is the authority on every public-facing member of the Angular libraries.
 
+  ## We welcome feedback! 
 
-  Use the [Angular Github repo](https://github.com/angular/angular) to report issues with **Angular** itself.
+  - Use the [site Github repo](!{_ngDocRepoURL}) for **documentation** issues and pull requests.
```
Comment (kwalrath) [inline public/docs/ts/latest/guide/index.jade]: Github -> GitHub

(Global. Not introduced in this PR, but while you're there...)

File: public/docs/ts/latest/guide/index.jade [hunk 2]
```diff
@@ -22,7 +21,7 @@ table(width="100%")
   col(width="15%")
   col
   tr(style=top)
-    td <b>QuickStart</b>
+    td <b><a href="../quickstart">QuickStart</a></b>
```
Comment (kwalrath) [inline public/docs/ts/latest/guide/index.jade]: It's a little weird to have links on QuickStart but not on Guide & Advanced... But it *is* definitely handy to have these links, especially for those times when the window is too narrow to display the sidenav.

This is a case where having everything in the guide/ directory is coming back to bite us.

Maybe we should try this out and see if anyone objects?

File: public/docs/dart/latest/api/index.jade [hunk 3]
```diff
@@ -1,9 +1,12 @@
+include ../_util-fns
+
 :marked
   > **Known issues:** The angular.io issue tracker contains [all known
```
Comment (kwalrath) [inline public/docs/dart/latest/api/index.jade]: angular.io -> Angular

File: public/docs/ts/latest/guide/index.jade (hunk 1 above)
Comment (chalin) [inline public/docs/ts/latest/guide/index.jade]: Fixed

File: public/docs/ts/latest/guide/index.jade (hunk 2 above)
Comment (kwalrath) [inline public/docs/ts/latest/guide/index.jade]: No, I don't think a self-link is helpful. We could maybe briefly mention in the text that that's where they are.

Comment (chalin) [inline public/docs/ts/latest/guide/index.jade]: (I was thinking about that, but wasn't sure how to adjust the text.)

**Pull request merged**